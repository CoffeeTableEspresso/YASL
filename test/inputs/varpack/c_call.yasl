



let a, let b = collections.set(1, 2, 3)


echo a
echo b
