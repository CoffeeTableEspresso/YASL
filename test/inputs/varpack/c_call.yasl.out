set(2, 1, 3)
undef
